projects/Mallard/src/services/errors.ts

```diff
@@ -40,11 +47,20 @@ class ErrorServiceImpl implements ErrorService {
         })
     }
 
-    private handleConsentUpdate(hasConsent: GdprSwitchSetting) {
-        this.hasConsent = hasConsent
-        if (hasConsent === false || hasConsent === null) return
+    private async handleConsentUpdate(hasConsent: GdprSwitchSetting) {
+        this.hasConsent = hasConsent === true
+        console.log('setting consent: ', this.hasConsent)
```

Comment:
maybe this isn't needed? I'm not sure about leaving console.logs in the client side code. Sometimes can be useful to see stuff when running the app though

Reply:
As I mentioned above:

```
I have been informed by @jimhunty that these logs aren't available in prod build. So it should be fine and help us monitor/debug things locally. 
```